Vagrantfile

```diff
@@ -77,22 +78,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     provider.video_type = "qxl"
   end
 
-  config.vm.define("el8", primary: true) do |vm_config|
-    vm_config.vm.box = "centos.cloud/centos8s"
-    vm_config.vm.box_url = "https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-Vagrant-8-latest.x86_64.vagrant-libvirt.box"
-    vm_config.vm.host_name = "candlepin-el8.example.com"
-
-    # Vagrant allows to create a forwarded port mapping which allows access to a specific port
-    # within the guest machine from a port on the host machine.
-    #
-    # Uncomment these lines to forward the Candlepin standard dev ports to this guest machine.
-    # vm_config.vm.networking "forwarded_port", protocol: "tcp", guest: 8080, host: 8080
-    # vm_config.vm.networking "forwarded_port", protocol: "tcp", guest: 8443, host: 8443
-    vm_config.vm.provision "shell", inline: "dnf update -y dnf ca-certificates"
-    configure_ansible_provisioning(vm_config)
-  end
-
-  config.vm.define("el9", autostart: false) do |vm_config|
+  config.vm.define("el9", primary: true) do |vm_config|
```

Comment:
nitpick: if only one vm is specified in vagrantfil there is no need to set the primary machine